README.md

```diff
@@ -1 +1,170 @@
-# ibm_cloud_tower_deploy
+Create and Destroy IBM Cloud Instances
+=======================================
+
+Tower CPT was broken due to increase in the disk IOPS requirements in Ansible Tower v4.0.0. So it made us to migrate from in-house VMs to IBM cloud VSI.
+
+We believe there are two use cases associated with this repo and these are our primitive attempts to describe it.
+1. Looking for virtual cloud instances (OR)
+2. Looking for Tower deployment on faster disks
+
+Looking for virtual cloud instances ??
+------------------------------------
+
+Install Ansible modules collection for IBM cloud
+
+    ansible-galaxy collection install awx.awx
```

Comment:
it should be:

ansible-galaxy collection install ibm.cloudcollection

Reply:
my bad. thanks